integration-tests/src/test/java/oracle/kubernetes/operator/utils/LoadBalancer.java

```diff
@@ -284,19 +292,49 @@ private void createVoyagerIngress() throws Exception {
         .append(" --set ")
         .append("voyager.webPort=")
         .append(lbMap.get("loadBalancerWebPort"));
-
     LoggerHelper.getLocal().log(Level.INFO, "createVoyagerIngress() Running " + cmd.toString());
-    executeHelmCommand(cmd.toString());
+    
+    String returnStr = null;
+    int i = 0;
+    // Wait max 300 seconds
+    while (i < maxIterationsPod) {
```

Comment:
adding loop for helm install will not help as helm install is not failing, it's just that the release deployment is taking time. helm install command will return without errors even though the deployment is not fully complete. See if checking the status of the release will help, it should be DEPLOYED or see if helm install command have an option to return only after deployment is complete.